VMAccess/vmaccess.py

```diff
@@ -127,6 +131,19 @@ def update():
     hutil.do_exit(0, 'Update', 'success', '0', 'Update Succeeded')
 
 
+# Previous versions of VMAccessForLinux set ChallengeResponseAuthentication to yes,
+# which they should not.  Version 1.4.5.0 forcefully resets the value back to 'no'.
```

Comment:
@boumenot Or even more simple: "Verison 1.4.5.0 reset the value back to 'no'." Adverbs are so unnecessary, most of the time...said the Grammar Maven.